sfs/mono/source.py

```diff
@@ -82,7 +82,9 @@ def point(omega, x0, grid, c=None):
     grid = util.as_xyz_components(grid)
 
     r = np.linalg.norm(grid - x0)
-    return 1 / (4*np.pi) * np.exp(-1j * k * r) / r
+    # `r` can get `0`, which is fine in this case
+    with np.errstate(invalid='ignore', divide='ignore'):
+        return 1 / (4*np.pi) * np.exp(-1j * k * r) / r
```

Comment:
We could probably also be more obvious here and put the numerator in a temporary variable (before opening the `with` statement).

The we could have only the critical division within the `with` statement.

```python
numerator = np.exp(-1j * k * r) / (4*np.pi)
with np.errstate(invalid='ignore', divide='ignore'):
    return numerator / r
```

Would this make sense?

Reply:
If we do it like this, we probably don't even need a separate comment explaining it?

Reply:
Good idea, then the `invalid='ignore'` should also be not needed.